config/300-build.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
```

Comment:
Currently, I believe the idea is to have Tasks create builds from the buildSpec field in a Task.  It is unclear if installing knative/build should be a dependency for knative/build-pipeline or we should import pieces of knative/build in knative/build-pipeline (build controller, etc.) and manage those pieces that way.  I am open to either.  Currently, the build CRD is in config/ as the installation method is `ko apply -f config` and Build CRD is a dependency for the Task CRD.

Reply:
I think it would be better as a dependency, similar to what serving has. This way we would have less duplication and can easily pick up changes in the build project

Reply:
knative/build is now a requirement to install knative/build-pipeline